src/wireviz/Harness.py

```diff
@@ -97,8 +99,17 @@ def create_graph(self) -> Graph:
                      html_line_breaks(connector.subtype),
                      f'{connector.pincount}-pin' if connector.show_pincount else None,
                      connector.color, '<!-- colorbar -->' if connector.color else None],
-                    '<!-- connector table -->' if connector.style != 'simple' else None,
-                    [html_line_breaks(connector.notes)]]
+                    '<!-- connector table -->' if connector.style != 'simple' else None]
+            if connector.additional_components:
+                rows.append(["Additional components"])
+                for extra in connector.additional_components:
+                    qty = extra.qty * connector.get_qty_multiplier(extra.qty_multiplier)
+                    if(self.mini_bom_mode):
+                        id = self.get_bom_index(extra.description, extra.unit, extra.manufacturer, extra.mpn, extra.pn)
+                        rows.append(component_table_entry(f'{id} ({extra.type.capitalize()})', qty, extra.unit))
```

Comment:
Personally, I would prefer some prefix to the BOM id number in the component table for increased readability, 
e.g. `'#'`, `'No '` or `'BOM Id '`, but other users might disagree (e.g. also US/UK differences), and I suggest supporting an optional string value in the `mini_bom_mode` parameter in addition to the current True and False values. I therefore suggest inserting this code just below the current assignment of the `id` variable:
```py
if isinstance(self.mini_bom_mode, str):
    id = f'{self.mini_bom_mode}{id}'
```
The `if self.mini_bom_mode:` above will still work as intended for both `bool` and `str` values.